oak/server/rust/oak_runtime/src/node/wasm.rs

```diff
@@ -85,7 +85,7 @@ struct WasmInterface {
 impl WasmInterface {
     /// Generate a randomized handle. Handles are random to prevent accidental dependency on
     /// particular values or sequence. See https://github.com/project-oak/oak/pull/347
-    fn allocate_new_handle(&mut self, channel: ChannelEither) -> AbiHandle {
+    fn allocate_new_handle(&mut self, channel: ChannelEitherOwned) -> AbiHandle {
```

Comment:
Would it make sense to introduce this "randomized" handle concept to the ABI interface? I don't think it's specific to Wasm.